bindings/rust/wasmedge-sys/examples/store_get_named_module.rs

```diff
@@ -62,10 +62,12 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     // register a wasm module from a in-memory wasm bytes.
     let extern_module = Loader::create(Some(&config))?.from_bytes(wasm_bytes)?;
     Validator::create(Some(&config))?.validate(&extern_module)?;
-    executor.register_named_module(&mut store, &extern_module, "extern")?;
+    let instance = executor.register_named_module(&mut store, &extern_module, "extern")?;
 
     // get the module named "extern"
-    let instance = store.module("extern")?;
+    // let instance = store.module("extern")?;
```

Comment:
Not sure why we need to keep this comment?

Reply:
Accept. I'll refactor this part. Thanks!